lib/src/main/java/com/auth0/jwt/JWTCreator.java

```diff
@@ -261,6 +275,7 @@ public String sign(Algorithm algorithm) throws IllegalArgumentException, JWTCrea
                 throw new IllegalArgumentException("The Algorithm cannot be null.");
             }
             headerClaims.put(PublicClaims.ALGORITHM, algorithm.getName());
+            headerClaims.put(PublicClaims.TYPE, "JWT");
```

Comment:
just saw this, but why the `PublicClaims.TYPE`, `PublicClaims.ALGORITHM`? isn't this making the code harder to understand? it's a jwt lib, the field is `typ` so I would have expected something like `PublicClaims.TYP`
At least me, as an end-user, would not be really confident and would have to check the code to see if it's the actual value I'm looking for.